build.sbt

```diff
@@ -251,6 +251,13 @@ lazy val twirl = http4sProject("twirl")
   .enablePlugins(SbtTwirl)
   .dependsOn(core, testing % "test->test")
 
+lazy val awsSigner = libraryProject("aws-signer")
```

Comment:
Would `aws` make more sense than `aws-signer`, in case we add more integrations?  Perhaps those would have to depend on the aws-sdk, whereas this does not, so maybe I'm wrong. :thinking:

Reply:
I think in terms of modularity, I'd agree that `aws` might depend on aws-sdk, and in fact might also depend on this module.

I think it makes sense as-is